src/advanced/service-interaction.md

```diff
@@ -0,0 +1,218 @@
+# Service Interaction
+
+[Exonum services](../architecture/services.md) may interact
+with each other using a set of APIs. This article dives
+into details how different kinds of interaction can be implemented
+and used.
+
+!!! warning
+    Interaction among services is considered an experimental feature
+    as of Exonum 1.0. Its interfaces and supported features may change
+    in the future.
+    Additionally, support of interaction may be limited in some environments,
+    e.g., for [Java services](../glossary.md#java-service).
+
+## Problem Overview
+
+The problem solved by the techniques described in this article
```

Comment:
IMHO this introduction sentence is an example of non-necessary officialese.

I'd rephrase it to be simpler, like:


> Given multiple services in the blockchain, some of them may want to interact with other services.
The interaction can include either reading the data of other service, or modifying its data using a service public interface.